scripts/gh-predeploy.js

```diff
@@ -271,15 +309,79 @@ function prepareToDeployPlugins(done) {
 
 
 /**
- * Convert json to front matter YAML config.
- * @param {!Object} json The json config.
- * @return {string} The front matter YAML config.
+ * Inject nav bar HTML for a specific example at the beginning of the body.
+ * @param {string} inputString The initial page HTML, as a string.
+ * @param {!Object} packageJson The contents of the example's package.json.
+ * @param {string} pageRoot The location of the example's files relative to the root of
+ *     the repository.
+ * @param {string} title The title to display in the nav bar.
+ * @returns {string} The modified contents of the page, as a string.
  */
-function buildFrontMatter(json) {
-  return `---
-${yaml.stringify(json)}
----
-`;
+function injectExampleNavBar(inputString, packageJson, pageRoot, title) {
+  // Build up information from package.json.
+  let description = packageJson.blocklyDemoConfig.description ?
+      `<div class="subtitle">${ packageJson.blocklyDemoConfig.description}</div>` : ``;
+  let codeLink = `https://github.com/google/blockly-samples/blob/master/${pageRoot}`;
+
+  const pages = packageJson.blocklyDemoConfig.pages;
+  const tabString = pages ? createExampleTabs(pageRoot, pages) : '';
+  let baseurl = '/blockly-samples';
+  // Assemble that information into a nav bar and tabs for getting to linked
+  // example pages.
+  let navBar = `
+  <!-- NAV BAR -->
+  <nav id="toolbar">
+    <a href="${baseurl}" id="arrow-back">
+      <i class="material-icons">close</i>
+      <img src="https://blocklycodelabs.dev/images/logo_knockout.png" class="logo-devs"
+        alt="Blockly logo" />
```

Comment:
The primary purpose of the ALT attribute is to provide text content for screen readers.  At the moment that 'a' link says "close Blockly logo", which isn't very clear as to what it does.  We can't change tho word "close" since it's a font.  But we can change the following ALT text.  Maybe change to "Blockly sample"?